test/index.js

```diff
@@ -411,11 +411,17 @@ describe('isPrecompiled', function () {
     assert.equal(ethUtils.isPrecompiled('0000000000000000000000000000000000000002'), true)
     assert.equal(ethUtils.isPrecompiled('0000000000000000000000000000000000000003'), true)
     assert.equal(ethUtils.isPrecompiled('0000000000000000000000000000000000000004'), true)
+    assert.equal(ethUtils.isPrecompiled('0000000000000000000000000000000000000005'), true)
+    assert.equal(ethUtils.isPrecompiled('0000000000000000000000000000000000000006'), true)
+    assert.equal(ethUtils.isPrecompiled('0000000000000000000000000000000000000007'), true)
+    assert.equal(ethUtils.isPrecompiled('0000000000000000000000000000000000000008'), true)
+    assert.equal(ethUtils.isPrecompiled(Buffer.from('0000000000000000000000000000000000000001', 'hex')), true)
```

Comment:
```js
for (let i = 1; i < 9; ++i) {
  assert.equal(ethUtils.isPrecompiled(new BN(i).toString(10, 40)), true)
}
```

Reply:
This is definitely more elegant, thanks for the suggestion. Unfortunately it kills the browser tests somehow due to Buffer usage, just tested it, and I didn't want to go deeper in debugging here, think this would be not worth it in this case. Is it ok to leave it as is?

Also, can you approve, If you are ok with both explanations?

Reply:
Yea, this is ok let it as is.

Reply:
Cool, thanks.